health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/NewCampaignCreate/AppModule.js

```
@@ -0,0 +1,61 @@
+    }
+  );
+
+
+  return (
+    <>
+      <div>
+        <HeaderComponent className="campaign-header-style">{t(`HCM_CHOOSE_MODULE`)}</HeaderComponent>
+      </div>
+      <div className="modules-container">
+        {modulesData?.map((module, index) => (
+          <Card className={"module-card"}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)